data/beaconrestapi/src/integration-test/java/tech/pegasys/teku/beaconrestapi/v1/debug/GetStateIntegrationTest.java

```diff
@@ -65,7 +65,7 @@ public void shouldGetStateAsOctetStream() throws IOException {
 
   @Test
   public void shouldRejectAltairStateRequestsForJson() throws IOException {
-    startRestAPIAtGenesis(SpecMilestone.ALTAIR);
+    startRestAPIAtGenesis(SpecMilestone.ALTAIR); // TODO check, should it not handle ALTAIR?
     final Response response = get("head");
     assertThat(response.code()).isEqualTo(SC_BAD_REQUEST);
```

Comment:
Does this mean `GetState` should not handle ALTAIR?

Reply:
Hrm... im less worried about v1... it'll handle altair but i've also just put up a PR to remove it from the spec...

Reply:
This integration test fails, as it seems to want ALTAIR to give a bad request.

Reply:
oh - so that's your answer :)

Reply:
Not sure if this helps but we shouldn't be able to get an Altair state as JSON from this API but should be able to get Altair states as SSZ.

Reply:
in that case it should only list the PHASE0 definition, wont allow anything else..

Reply:
Ah okay that makes things tricky